lib/utils/__tests__/isStandardSyntaxKeyframesName.test.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const isStandardSyntaxKeyframesName = require('../isStandardSyntaxKeyframesName');
+
+describe('isStandardSyntaxKeyframesName', () => {
+	it('standard name', () => {
+		expect(isStandardSyntaxKeyframesName('@keyframes slidein {}')).toBeTruthy();
```

Comment:
```suggestion
		expect(isStandardSyntaxKeyframesName('slidein')).toBeTruthy();
```

As the util is checking the [keyframes-name](https://drafts.csswg.org/css-animations/#typedef-keyframes-name), and not the whole `@keyframe`.

We'll need to update the other tests too.